src/app/insite-nest-plot/insite-nest-plot.component.ts

```diff
@@ -65,7 +65,7 @@ export class InsiteNestPlotComponent implements OnInit {
     var from = Math.max(this.from-50, 0);
     var to = Math.max(this.to+50, 0);
     let params = new HttpParams().set('from', from.toString()).set('to', to.toString());
-    this.http.get(this.url + ':8000/spikes', {params: params}).subscribe(res => {
+    this.http.get(this.url + ':8080/spikes', {params: params}).subscribe(res => {
```

Comment:
I set the port 8000 because from arguments doesn't work. Requesting on port 8080 it always receives spikes from 0ms. You can check it with: `curl localhost:8000/spikes?from=9500` and  `curl localhost:8080/spikes?from=9500`.
The problem should be resolved in insite module.

Reply:
Oh good to know, that is a bug on our side, we will fix that!